File: ApiSuvesaPos/Negocio/Modelo/inventario/movimientobodega/movimientosbodega_detalle.vb
Namespace Modelo
    Public Class movimientosbodega_detalle
        Public Property Id_Detalle As Integer
        Public Property Boleta_Movimiento As Long
        Public Property Codigo As String
        Public Property CodArticulo As String
        Public Property Descripcion As String
        Public Property Cantidad As Decimal
        Public Property Tipo_Entrada As Boolean
        Public Property Tipo_Salida As Boolean
        Public Property Numero As String
        Public Property ExistenciaForzada As Decimal
        Public Property ExistVeteForzada As Decimal
        Public Property Serie As New List(Of serie) 'las series que salen o que entran
    End Class
End Namespace

File: ApiSuvesaPos/ApiSuvesaPos/Modelo/inventario/movimientobodega/movimientosbodega_detalle.cs
namespace Modelo
{
    public class movimientosbodega_detalle
    {
        public int Id_Detalle { get; set; }
        public long Boleta_Movimiento { get; set; }
        public string Codigo { get; set; }
        public string CodArticulo { get; set; }
        public string Descripcion { get; set; }
        public decimal Cantidad { get; set; }
        public bool Tipo_Entrada { get; set; }
        public bool Tipo_Salida { get; set; }
        public string Numero { get; set; }
        public decimal ExistenciaForzada { get; set; }
        public decimal ExistVeteForzada { get; set; }
        //public List<serie> Serie { get; set; } = new List<serie>(); // las series que salen o que entran
    }
}

